### PR TITLE
MEN-6671: Couple of integration tests fixes

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

File: tests/tests/test_mtls.py [hunk 1]
```diff
@@ -133,15 +133,13 @@ def hsm_get_key_uri(self, pin, ssl_engine_id, device):
             + '\nMODULE_PATH = /usr/lib/softhsm/libsofthsm2.so\ninit = 0\n" >> /etc/ssl/openssl.cnf'
         )
         device.run(
-            'sed -i.backup -e "/\\[Service\\]/ a Environment=SOFTHSM2_CONF=/softhsm/softhsm2.conf" /lib/systemd/system/%s.service'
-            % device.get_client_service_name()
+            'sed -i.backup -e "/\\[Service\\]/ a Environment=SOFTHSM2_CONF=/softhsm/softhsm2.conf" /lib/systemd/system/mender-authd.service'
```
Comment (oleorhagen) [inline tests/tests/test_mtls.py:136]: This now, as far as I can tell, needs to be done for both services :thinking:

File: tests/tests/test_preauth.py [hunk 2]
```diff
@@ -239,7 +237,7 @@ def substitute_id_data(device, id_data_dict):
     def restart(device):
         """Restart the mender service."""
 
-        device.run("systemctl restart %s.service" % device.get_client_service_name())
+        device.run("systemctl restart mender-updated")
```
Comment (oleorhagen) [inline tests/tests/test_preauth.py:240]: We shuuure this should not be `auth`?

**Review by oleorhagen — APPROVED** [review]
Real nice - thx!

:medal_military:

File: tests/tests/test_preauth.py (hunk 2 above)
Comment (lluiscampos) [reply tests/tests/test_preauth.py:240]: This test actually passes, so it needs to be right.

However I share with you the hunch that it _should_ be `mender-authd`. Which brings me to the realization that we actually don't need to restart the service at all: `mender-auth` will pick up the updated id script once it is modified on disk. I'll remove the restart for clarity.

File: tests/tests/test_mtls.py (hunk 1 above)
Comment (lluiscampos) [reply tests/tests/test_mtls.py:136]: I'll do it for good measure. This test is not passing yet.

File: tests/tests/test_mtls.py [hunk 3]
```diff
@@ -183,8 +181,7 @@ def common_test_mtls_enterprise(self, env, algorithm=None, use_hsm=False):
                 remote_path="/var/lib/mender",
             )
 
-        client_service_name = env.device.get_client_service_name()
-        env.device.run("systemctl stop %s" % client_service_name)
+        env.device.run("systemctl stop mender-updated")
```
[the remark this answers is not in the document]
Comment (lluiscampos) [reply tests/tests/test_mtls.py:184]: Related to the comment above, I'll do both.

**Comment by lluiscampos** [comment]
@oleorhagen I would appreciate your eyes again as I had to fix few things in the rework :+1:

**Comment by lluiscampos** [comment]
This PR resulted in more failures than our nightly reference. So refrain further review from now.

I submitted the fix for test_configuration separately (this test actually passed here :tada: ). See:
* https://github.com/mendersoftware/integration/pull/2477

**Comment by lluiscampos** [comment]
Superseded by #2503

**Pull request closed without merging**